flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -37,6 +37,18 @@
 			.withDeprecatedKeys(WebOptions.ADDRESS.key(), ConfigConstants.DEFAULT_JOB_MANAGER_WEB_FRONTEND_ADDRESS.key())
 			.withDescription("The address that the server binds itself.");
 
+	/**
+	 * The port range that the server could bind itself to.
+	 */
+	public static final ConfigOption<String> BIND_PORT =
+		key("rest.bind-port")
+			.defaultValue("8081")
+			.withDeprecatedKeys("rest.port", WebOptions.PORT.key(), ConfigConstants.JOB_MANAGER_WEB_PORT_KEY)
```

Comment:
you could move `rest.port` into a static field that is referenced here and in `PORT`, then this key is coupled more tightly to the actual option.

Reply:
With fallback keys being a thing now we could use that for `rest.port`. It's convenient for local setups to being able to set one port and have the server/client work with that.

Reply:
@zentol thanks for your suggestion. I have defined a private static filed named `REST_PORT_KEY` in `RestOptions` class, because I think this key is just referenced for REST service, so I do not define it `ConfigConstants` and it seems `ConfigConstants` is the legacy class. If you think `ConfigConstants` is a better choice, I also would like to change this field's location.